core/tests/protractor_desktop/userJourneys.js

```diff
@@ -240,9 +240,15 @@ describe('Site language', function() {
     await general.openEditor(firstExplorationId, false);
 
     // Spanish is still selected.
-    var placeholder = await element(by.css('.protractor-test-float-form-input'))
-      .getAttribute('placeholder');
-    expect(placeholder).toEqual('Ingresa un número');
+    var placeholderElement = await element(
+      by.css('.protractor-test-float-form-input'));
+    await waitFor.visibilityOf(
+      placeholderElement, 'Placeholder Element taking too long to appear');
+    await waitFor.elementAttributeToBe(
+      placeholderElement, 'placeholder', 'Ingresa un número',
+      'Placeholder text taking too long to change from English to Spanish');
```

Comment:
In my investigation I found that English version of the placeholder text changed to Spanish version after a few seconds. Hence waited with `elementAttributeToBe`

Reply:
@kevintab95 is this language change expected, or is it a bug in the frontend code?

Reply:
Just as an FYI, I tried to perform the changes manually in the test server. Here's what I did:

1. Create a numeric input exploration.
2. Changed the site's language from English to Spanish.
3. Navigated to the exploration created with the exploration id.

After this I saw that the english version of "Type a number" changed to Spanish after a few seconds.
Here's the video of the same.

https://user-images.githubusercontent.com/69013647/131332882-7e6342e5-2f54-4a8b-8ee7-0b93a61622d2.mp4

Reply:
Yup the language change is expected.